biothings/web/analytics/channels.py

```diff
@@ -1,83 +1,99 @@
+import aiohttp
+import asyncio
 import certifi
+import logging
 import orjson
-from tornado.httpclient import HTTPRequest
 
 from biothings.web.analytics.events import Event, Message
 
-
 class Channel:
```

Comment:
We can use the [abc module](https://docs.python.org/3/library/abc.html) to force the usage of this class as an interface